tests/test_prior.py

```diff
@@ -46,6 +49,41 @@ def test_log_prior_with_no_prior():
     assert param.log_prior().numpy() == 0.0
 
 
+def test_log_prior_for_uniform_prior():
+    """
+    A parameter with a uniform prior should have uniform log-prior,
```

Comment:
Can you clarify what you mean here? It doesn't make sense to me how a uniform prior should be a uniform log-prior...

Reply:
Sorry bad wording - has been updated now (log-prior here referred to the function call log(p) rather than a log distribution.)